decidim-comments/app/frontend/comments/add_comment_form.component.jsx

```diff
@@ -194,7 +194,7 @@ export class AddCommentForm extends Component {
             className={okButtonClassName}
             onClick={() => this.setState({ alignment: 1 })}
           >
-            <Icon name="icon-thumb-up" />
+            <Icon iconExtraClassName="" name="icon-thumb-up" />
```

Comment:
As discussed offline with @lastpotion the `iconExtraClassName` has a default prop value so we need to override it here.